docs/development.md

```diff
@@ -65,9 +65,16 @@ The details on how to run them locally can be found [here (for local Datastore e
 
 If you have access to Docker, we have a Docker compose definition to run those services:
 
+```sh
+docker compose up -d
+
+```
+If the following command does not work, you may not have the updated version (v2). So use this instead:
```

Comment:
```suggestion
If the above command does not work, you may not have the updated version (v2) of Docker. You may want to try this instead:
```

Reply:
To me, "... following command..." seems to refer to the command **after** this sentence (i.e. `docker-compose`), which seems out of order. I think it would be clearer to use "... above command..."